tests/installation/bootloader_zkvm.pm

```diff
@@ -81,11 +86,13 @@ sub run {
 
     if (!get_var("BOOT_HDD_IMAGE") or (get_var('PATCHED_SYSTEM') and !get_var('ZDUP'))) {
         if (check_var("VIDEOMODE", "text")) {
-            wait_serial("run 'yast.ssh'", 300) || die "linuxrc didn't finish";
-            select_console("installation");
-            script_run("clear");
-            # If libyui REST API is used, we set it up in installation/setup_libyui
-            enter_cmd("TERM=linux yast.ssh") unless get_var('YUI_REST_API');
+            unless (get_var('AGAMA')) {
```

Comment:
there is no yast, so the else below is not reachable either, right? you could just `return if get_var('AGAMA')` wdyt?

Reply:
Yes, updated.